src/core/config_mod.F90

```diff
@@ -227,9 +231,129 @@ SUBROUTINE Config_Simulation( ConfigInput, Config, RC )
       Config%SimulationName = TRIM( v_str )
 
 
+      ! Error check simulation name
+      Sim = To_UpperCase( TRIM( Config%SimulationName ) )
+      IF ( TRIM(Sim) /= 'AEROSOL' ) THEN
+           errMsg = Trim( Config%SimulationName) // ' is not a'            // &
+                    ' valid simulation. Supported simulations are:'           // &
+                    ' aerosol.'
+           CALL CC_Error( errMsg, RC, thisLoc )
+           RETURN
+      ENDIF
+
+      ! Set simulation type flags in Config
+      Config%ITS_AN_AEROSOL_SIM   = ( TRIM(Sim) == 'AEROSOL'               )
```

Comment:
Not sure what the point to these are.... this looks like a holdover from geos-chem